### PR TITLE
Infrastructure: Add eslint-html plugin

### PR DESCRIPTION
**Pull request by nschonni**
Allows linting and autofixing of the script tags inside the HTML files

### CODE REVIEW

**Comment by nschonni** [comment]
@zcorpan looks like the regression test token issue isn't fully resolved. Reading up some more from the error message https://github.community/t/github-actions-are-severely-limited-on-prs/18179/6, but you may as well disable it for now, so it doesn't turn other PRs red

**Comment by nschonni** [comment]
FYI, people that are collaborators shouldn't have the failure, just people who try and fork and contribute, so maybe not too much of an issue

**Comment by zcorpan** [comment]
@nschonni it is an issue IMO, so I've disabled the workflow again. Tracking in https://github.com/w3c/aria-practices/issues/1811#issuecomment-959512379

If you amend the last commit with no changes and force push the CI checks should re-run without the regression test coverage report check.

**Comment by nschonni** [comment]
@zcorpan green again after the rebase

File: examples/listbox/listbox-collapsible.html [hunk 1]
```diff
@@ -304,8 +304,10 @@ <h2 id="sc_label">HTML Source Code</h2>
     <div id="sc_start_sep" role="separator" aria-labelledby="sc_start_sep sc_label" aria-label="Start of "></div>
     <pre><code id="sc"></code></pre>
     <div id="sc_end_sep" role="separator" aria-labelledby="sc_end_sep sc_label" aria-label="End of"></div>
-    <script>sourceCode.add('sc', 'ex', 'ex_label', 'css_js_files')</script>
-    <script> sourceCode.make() </script>
+    <script>sourceCode.add('sc', 'ex', 'ex_label', 'css_js_files');
+</script>
+    <script>sourceCode.make();
+ </script>
```
Comment (zcorpan) [inline examples/listbox/listbox-collapsible.html:310]: Can you fix this formatting? (Also in other files.)

**Review by zcorpan — CHANGES_REQUESTED** [review]

File: examples/listbox/listbox-collapsible.html (hunk 1 above)
Comment (nschonni) [reply examples/listbox/listbox-collapsible.html:310]: Unfortunatly, that is how it auto-fixes it. I reverted some of them, but then they change back after

Comment (zcorpan) [reply examples/listbox/listbox-collapsible.html:310]: What happens if you include a newline after `<script>`?

Comment (nschonni) [reply examples/listbox/listbox-collapsible.html:310]: Think it's a known issue https://github.com/BenoitZugmeyer/eslint-plugin-html/issues/131

Comment (zcorpan) [reply examples/listbox/listbox-collapsible.html:310]: In files where there is a newline after the start tag the indentation is OK. Can you add a newline after the start tag here and see what autofix does?
```suggestion
    <script>
      sourceCode.add('sc', 'ex', 'ex_label', 'css_js_files');
    </script>
    <script>
      sourceCode.make();
    </script>
```

File: examples/coding-template/Depricated-MultipleImplementationExample-Template.html [hunk 2]
```diff
@@ -268,7 +268,8 @@ <h3 id="sc1_label">Example 1: title of this variation of the implementation</h3>
       It renders the HTML in the preceding pre element with ID 'sc1'.
       If you change the ID of either the 'ex1' div or the 'sc1' pre, be sure to update the sourceCode.add function parameters.
     -->
-    <script>sourceCode.add('sc1', 'ex1')</script>
+    <script>sourceCode.add('sc1', 'ex1');
+</script>
```
Comment (zcorpan) [inline examples/coding-template/Depricated-MultipleImplementationExample-Template.html:272]: Newline + indentation 🙂 also below

**Review by zcorpan — APPROVED** [review]

**Pull request merged**